x-pack/tasks/build.ts

```diff
@@ -72,13 +72,17 @@ async function copySource() {
           '**/node_modules/**',
           '**/public/**/*.{js,ts,tsx,json,scss}',
           '**/{test,__tests__,__mocks__,__snapshots__,__fixtures__,__jest__,cypress,fixtures}/**',
+          '**/dev_docs}/**',
```

Comment:
@jbudz this looks wrong

Reply:
thanks - I reverted the whole file with https://github.com/elastic/kibana/pull/137689/commits/cc6cf1ef183ee79cc321d663c157a31c91a3cf13.  This PR should just have the screenshot changes now.